uefi-test-runner/examples/timestamp.rs

```diff
@@ -0,0 +1,60 @@
+// ANCHOR: all
```

Comment:
Unless we have evidence of an actual UEFI implementation that supports the timestamp protocol, let's drop this example.

Reply:
I run the test in VMware which is UEFI 2.7 firmware. You could see those output:
![image](https://github.com/rust-osdev/uefi-rs/assets/12370628/d761cbd5-c5e4-4804-b6c5-0792dac49497)

Reply:
Oh nice :)